apache/crontabFile

```diff
@@ -0,0 +1,37 @@
+# Edit this file to introduce tasks to be run by cron.
+# 
+# Each task to run has to be defined through a single line
+# indicating with different fields when the task will be run
+# and what command to run for the task
+# 
+# To define the time you can provide concrete values for
+# minute (m), hour (h), day of month (dom), month (mon),
+# and day of week (dow) or use '*' in these fields (for 'any').# 
+# Notice that tasks will be started based on the cron's system
+# daemon's notion of time and timezones.
+# 
+# Output of the crontab jobs (including errors) is sent through
+# email to the user the crontab file belongs to (unless redirected).
+# 
+# For example, you can run a backup of all your user accounts
+# at 5 a.m every week with:
+# 0 5 * * 1 tar -zcf /var/backups/home.tgz /home/
+# 
+# For more information see the manual pages of crontab(5) and cron(8)
+# 
+# m h  dom mon dow   command
+# Every two minutes
+# 1-59/2 * * * *
+# Every five minutes
+# */5 * * * *
+# Every 30 minutes
+# */30 * * * *
+
+* * * * * /usr/local/bin/php  /var/www/html/app/console mautic:leadlists:update
+* * * * * /usr/local/bin/php /var/www/html/app/console mautic:campaigns:update
+* * * * * /usr/local/bin/php /var/www/html/app/console mautic:campaigns:trigger
+* * * * * /usr/local/bin/php /var/www/html/app/console mautic:email:process 
+* * * * * php /var/www/html/app/console mautic:fetch:email 
```

Comment:
Can you make the PHP command consistent (either all as `php` or all with the full file path)?